airbyte-integrations/connectors/source-sumologic/source_sumologic/spec.json

```diff
@@ -0,0 +1,60 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/sumologic",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Sumologic Spec",
+    "type": "object",
+    "required": ["access_id", "access_key", "query"],
+    "additionalProperties": false,
+    "properties": {
+      "access_id": {
+        "type": "string",
+        "title": "Access ID",
+        "description": "Sumo Logic API Access ID.",
+        "airbyte_secret": true
+      },
+      "access_key": {
+        "type": "string",
+        "title": "Access Key",
+        "description": "Sumo Logic API Access Key.",
+        "airbyte_secret": true
+      },
+      "query": {
+        "type": "string",
+        "title": "Query",
+        "description": "Sumo Logic query string for the search job. You may need to escape certain characters."
+      },
+      "from_time": {
+        "type": "string",
+        "title": "From Time",
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}$",
+        "description": "UTC date and time in the format 2017-01-25T00:00:00. Any data before this date will not be replicated. Leave empty if you want to pull from the very start.",
+        "examples": ["2017-01-25T00:00:00"]
+      },
+      "to_time": {
+        "type": "string",
+        "title": "To Time",
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}$",
+        "description": "UTC date and time in the format 2017-01-25T00:00:00. Any data after this date will not be replicated. Leave empty if you want to pull util the most recent.",
+        "examples": ["2017-01-25T00:00:00"]
```

Comment:
Makes sense. The reason I included `to_time` is because this param required by sumo-logic search job API https://help.sumologic.com/APIs/Search-Job-API/About-the-Search-Job-API#query-parameters

I think I can hide the logic, not allow users to set it, just always auto-generate a to_time in the code.

Reply:
Addressed. `to_time` is now handled by default, not an option anymore.